_drafts/java-web-start.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Java Web Start
+categories: tech-issues
+---
+
+##Background
+
+Java Web Start was introduced in 2001 to allow applications to be launched through browsers or directly via the Java Network Launching Protocol (JNLP).
+
+Java Web Start Applications do not run inside the browser like Applets but still run in a restricted environment. Those restrictions can be removed by signing the applications with a trusted certificated. This has been encouraged since Java 7 update 21 to lower risks for desktop users and it has been re-inforced in follow-up updates.
+Two security changes to enhance authentication and authorization for Web Start (and Applets) were introduced back in January 2014 with Java 7 update 51.
+
+As of Java 7 update 51
+Rich Internet Applications must contain two things:
+1. Code signatures from a trusted authority.
+2. Manifest Attributes:
+   i. Permissions (required) indicating if the application should run within the sandbox or it requires full-permissions.
+   ii. Codebase (recommended): location of the hosted code.
+
+Due to vulnerabilities affecting Java plugins, users are now frequently recommended to disable Java, at least, in their favorite browser. Since 2013, browsers like Firefox, Google chrome just to mention a few have started to block plugins by default.
+The US Department of Homeland Security has been recommended since 2013 to disable Java in Web browsers unless it is absolutely necessary to run it.
+
+##What does it mean for desktop administrators?
+
+To deploy Java Web Start, you first need to get familiar with [Deployment Rule Sets](https://blogs.oracle.com/java-platform-group/entry/introducing_deployment_rule_sets).
+Administrators can then create a list of known-safe applications and manage compatibility between
```

Comment:
Only if the people deploying Java Web Start, and the Administrators, are the same people.